@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
+    <div class="host-network-table-container">
+      <UiTitle type="h4" class="header">
+        <slot>{{ $t('pifs') }}</slot>
+        <template #actions>
+          <UiButton size="medium" variant="secondary" accent="info" :left-icon="faArrowsRotate">
+            {{ $t('scan-pifs') }}
+          </UiButton>
+        </template>
+      </UiTitle>
+      <div v-if="isReady" class="table-container">
+        <UiQuerySearchBar class="table-query" @search="(value: string) => (filter = value)" />
```

Comment:
Same here, the class is not needed:
```suggestion
        <UiQuerySearchBar @search="(value: string) => (filter = value)" />
```